### ISSUE TITLE
Quick Tour - Add info on newline keybindings once they are live in 0.101

### ISSUE THREAD

**Issue by NotTheDr01ds** [nushell/nushell.github.io#1671]
From #1662 review, [this comment](https://github.com/nushell/nushell.github.io/pull/1662#discussion_r1872230873) - A reminder to add information on the newline keybindings once they go live with 0.101 in a few weeks.

**Comment by Kissaki** [comment]
The [Quick Tour section in question](https://www.nushell.sh/book/quick_tour.html#command-arguments-in-a-pipeline) has a tips block that mentions multi-line editability, how it is equivalent to in a single line, and then links to multi-line editing.

> See Also: [Multi-line Editing](https://www.nushell.sh/book/line_editor.html#multi-line-editing)

The section is already very filled with a lot of information. In my opinion, the link to multi-line editing is sufficient and we should not add REPL line break into that section specifically. I would close this issue as-is.

The [multi-line editing docs](https://www.nushell.sh/book/line_editor.html#multi-line-editing) do already mention

> 3. Manually insert a newline using Alt+Enter or Shift+Enter.

**Comment by NotTheDr01ds** [comment]
@Kissaki Agreed - I added them and forgot to close this.  Thanks for looking at it!
